README.md

```diff
@@ -96,31 +96,52 @@ Optional high-code extensions allow almost infinite customization in a modular w
 <img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/dashboard_examples.png" width="1300"/>
 </p>
 
-### Live demo
+## Packages
 
-<p align="left">
+|           Folder           |                                           Version                                           |                        Documentation                        |
+| :------------------------: | :-----------------------------------------------------------------------------------------: | :---------------------------------------------------------: |
+| [vizro-core](./vizro-core) |    [![PyPI version](https://badge.fury.io/py/vizro.svg)](https://badge.fury.io/py/vizro)    |    [Vizro Docs](https://vizro.readthedocs.io/en/stable/)    |
+|   [vizro-ai](./vizro-ai)   | [![PyPI version](https://badge.fury.io/py/vizro-ai.svg)](https://badge.fury.io/py/vizro-ai) | [Vizro-AI Docs](https://vizro-ai.readthedocs.io/en/latest/) |
+
+### vizro-core
+
+<p align="center">
 <a href="http://vizro.mckinsey.com/">
 <img border="0" src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/live_interactive_demo.png" width="525" height="296">  </a>
+<em>Live demo</em>
```

Comment:
added a new section called vizro-core with just moving live demo and all sorts of sections below `vizro-core`